packages/renderer/src/lib/configmaps-secrets/ConfigMapSecretList.spec.ts

```diff
@@ -42,8 +42,10 @@ test('Expect configmap empty screen', async () => {
   vi.mocked(states).kubernetesCurrentContextConfigMapsFiltered = writable<KubernetesObject[]>([]);
   vi.mocked(states).kubernetesCurrentContextSecretsFiltered = writable<KubernetesObject[]>([]);
   render(ConfigMapSecretList);
-  const noNodes = screen.getByRole('heading', { name: 'No configmaps or secrets' });
-  expect(noNodes).toBeInTheDocument();
+  await vi.waitFor(() => {
+    const noNodes = screen.getByRole('heading', { name: 'No configmaps or secrets' });
+    expect(noNodes).toBeInTheDocument();
```

Comment:
nit: I think that getByRole is failing if it's not in the document anyway (and I see the same lines being repeated over and over)

Reply:
Yes right, I see it everywhere in the tests, so I do the same, but they are effectively not necessary. I'll add an entry to the code guideline and make some cleanup, as part of other issues

Reply:
yes it's not blocking the merge